src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -295,50 +296,52 @@
 
 (defn message-content-wrapper
   "Author, userpic and delivery wrapper"
-  [{:keys [last-in-group?
-           identicon
-           from in-popover? timestamp-str
+  [{:keys [last-in-group? identicon from in-popover? timestamp-str
            deleted-for-me? pinned]
     :as   message} content {:keys [modal close-modal]}]
   (let [response-to (:response-to (:content message))]
-    [react/view {:style               (style/message-wrapper message)
-                 :pointer-events      :box-none
-                 :accessibility-label :chat-item}
-     (when (and (seq response-to) (:quoted-message message))
-       [quoted-message response-to (:quoted-message message)])
-     [react/view {:style          (style/message-body)
-                  :pointer-events :box-none}
-      [react/view (style/message-author-userpic)
-       (when (or (and (seq response-to) (:quoted-message message)) last-in-group? pinned)
-         [react/touchable-highlight {:on-press #(do (when modal (close-modal))
-                                                    (re-frame/dispatch [:chat.ui/show-profile from]))}
-          [photos/member-photo from identicon]])]
-      [react/view {:style (style/message-author-wrapper)}
-       (when (or (and (seq response-to) (:quoted-message message)) last-in-group? pinned)
-         [react/view {:style {:flex-direction :row :align-items :center}}
-          [react/touchable-opacity {:style    style/message-author-touchable
-                                    :disabled in-popover?
-                                    :on-press #(do (when modal (close-modal))
-                                                   (re-frame/dispatch [:chat.ui/show-profile from]))}
-           [message-author-name from {:modal modal}]]
-          [react/text
-           {:style               (merge
-                                  {:padding-left 5
-                                   :margin-top   2}
-                                  (style/message-timestamp-text))
-            :accessibility-label :message-timestamp}
-           timestamp-str]])
-       ;; MESSAGE CONTENT
-       ;; TODO(yqrashawn): wait for system message component to display deleted for me UI
-       (if deleted-for-me?
-         [react/view {:style {:border-width 2
-                              :border-color :red}}
-          content]
-         content)
-       [link-preview/link-preview-wrapper (:links (:content message)) false false]]]
-     ; delivery status
-     [react/view (style/delivery-status)
-      [message-delivery-status message]]]))
+    (if deleted-for-me?
+      [react/view {:style (style/deleted-for-me-message-body)}
+       [system-message/system-message
+        {:type :deleted-for-me :timestamp-str timestamp-str :non-pressble? true}]]
+      [react/view {:style               (style/message-wrapper message)
```

Comment:
Add deleted-for-me message at top of `message-content-wrapper`.
This is based on design.
Should this be handled in `style/message-wrapper` or `->message`?